packages/beachball-lock-update/package.json

```diff
@@ -24,7 +24,7 @@
     "src/index.js"
   ],
   "devDependencies": {
-    "@ni/eslint-config-javascript": "^3.1.0",
```

Comment:
Heads up that this package is being deleted in #797. @m-akinc do you plan to complete that PR today?

Reply:
I was waiting for Milan to deprecate the package, but I think I'll go ahead with submitting.